sale_coupon_mass_mailing/views/sale_coupon_program_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="sale_coupon_program_view_form_common_custom" model="ir.ui.view">
+        <field name="model">sale.coupon.program</field>
+        <field
+            name="inherit_id"
+            ref="sale_coupon.sale_coupon_program_view_form_common"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_count']/.." position="before">
```

Comment:
Always use the placeholder `oe_button_box` with position="inside" for not depending on existing buttons.